tests/test_main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from unittest import mock
+
+from app.main import cryptocurrency_action
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_is_not_substantive(mocked_prediction):
+    mocked_prediction.return_value = 3
```

Comment:
Add some boundary conditions, what if mocked.returned value = 1,05 * value passed in `cryptocurrency_action`?